web-app/src/lib/db/services/credit.service.ts

```diff
@@ -0,0 +1,105 @@
+import { CreditTransactionStatus, CreditTransactionType } from "@prisma/client";
+import { z } from "zod";
+
+import prisma from "@/lib/db/prisma";
+
+export async function getCreditBalance(userId: string): Promise<number> {
+  const creditBalance = await prisma.creditTransaction.aggregate({
+    where: { userId },
+    _sum: {
+      amount: true,
+    },
+  });
+
+  return Number(creditBalance._sum.amount ?? 0);
+}
+
+export async function creditActionSpend(
```

Comment:
Shouldn't it be `creditTransactionSpend`?

Reply:
Resolved https://github.com/masumi-network/sokosumi/pull/208/commits/c737666945b528fe98185b993b49143e90821578

Reply:
Nothing changed 🤷 are you sure you pushed the changes?

Reply:
Sorry linked the wrong commit it is in https://github.com/masumi-network/sokosumi/pull/208/commits/eabc84b4ca92ea9a794f04dcce229ba2c8fe6f27